graph/GRAPH.h

```diff
@@ -0,0 +1,120 @@
+#include<iostream>
+#include<vector>
+#include<queue>
+#include<list>
+#include<tuple>
+#include<set>
+
+
+#ifndef GRAPH_h
+#define GRAPH_h
+
+class graph
+{
+public:
+
+    vector<list<pair<int, int>>> g;
```

Comment:
As `using namespace std;` was removed, this won't work anymore.
You can also add `using std::vector;`, which applies to all the other `std::` members as well.